package.json

```diff
@@ -6,27 +6,35 @@
   "scripts": {
     "start": "webpack-dev-server",
     "build": "webpack --env.minify",
-    "test": "echo 'FOR TRAVIS CI ONLY PLZ USE YOUR OWN TESTS :)' && npm run build && echo 'SUCCESS!'"
+    "hyper-precompile": "browserify src/index.js -o bundle.pre.build.js -t [ babelify --presets [ es2015 ] ]",
+    "hyper-compile": "npm run hyper-precompile && browserify -t hyperxify -g uglifyify bundle.pre.build.js | uglifyjs > bundle.js",
+    "hyper-build": "npm run build && rm bundle.js && npm run hyper-compile && rm bundle.pre.build.js",
```

Comment:
Webpack builds an actual css file. Browserify bundles css into javascript and on huge apps this can get quite heavy, and to keep the API the same for the HTML file, I just decided to go this route.

1. Webpack builds both files
1. rm the js bundle
1. build temp file so that hyperxify can read everything (imports were breaking it)
1. then hyperxify the temp file
1. rm temp file
1. done 🎉